src/category/monad/writer.lean

```diff
@@ -148,7 +148,8 @@ export monad_writer_adapter (adapt_writer)
 section
 variables {ω ω' : Type u} {m m' : Type u → Type v}
 
-@[priority 100] -- see Note [lower instance priority]
+/- marked as nolint, because it generates metavariables during type class resolution -/
```

Comment:
Is this a problem? I suppose there aren't many relevant instances here to create bad searches.

Reply:
This instance is currently not dangerous. I looked at all the instances for these classes, and the search if finite, despite the metavariables. It might become problematic once we add recursive instances (see also my next comment).